OutOfSchool/OutOfSchool.WebApi/Startup.cs

```diff
@@ -111,6 +111,8 @@ public static void Configure(this WebApplication app)
         // Enable extracting token from QueryString for Hub-connection authorization
         app.UseMiddleware<AuthorizationTokenMiddleware>();
 
+        app.UseResponseCaching();
```

Comment:
double check if we have no extensions that set this up, as i think we've added some caching already in the project.